.github/workflows/e2e.yaml

```
@@ -138,6 +139,14 @@ jobs:
             target="test/e2e"
           fi
           make ${MAKE_PARAMETERS} CI=true "${target}"
+      - name: "Archive production artifacts"
+        if: ${{ always() }}
+        uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
+        with:
+          name: test-e2e-${{ env.E2E_RUN_NAME }}-${{ (github.event.pull_request.head.sha||github.sha) }}-${{ github.run_id }}
```

```suggestion
          name: test_e2e_${{ env.E2E_RUN_NAME }}-${{ (github.event.pull_request.head.sha||github.sha) }}-${{ github.run_id }}
```

to comform with op framework requirements